libs/core/src/lib/resizable-card-layout/resizable-card-item/resizable-card-item.component.scss

```diff
@@ -0,0 +1,23 @@
+@import "~fundamental-styles/dist/resizable-card-layout";
+
+.fd-resizable-card-layout__resize--vertical {
+    position: inherit;
+    z-index: 2;
+}
+
+.fd-resizable-card-layout__item {
+    display: inline-flex !important;
+    max-width: 100%;
+    max-height: 100%;
+    margin: 0 !important;
+    padding: 0.5rem !important;
```

Comment:
why are all rules important?

Reply:
i was over-riding values coming from fundamental-styles.
but now using "resizable-card-layout.css" directly. this can be removed once styles are released and version upgraded.